src/server/templates/target.js

```diff
@@ -55,6 +55,7 @@ export default ({
 		'BookBrainz – The Open Book Database'}</title>
 			<link rel='stylesheet' href='/stylesheets/bundle.css' />
 			<link rel='stylesheet' href='/stylesheets/style.css' />
+			<link rel='stylesheet' href='/stylesheets/external-services.css' />
```

Comment:
So I wasn't very clear in my instructions about how to add a new SCSS file, my bad !

Instead of processing it separately and adding it in target.js, the file needs to be imported in styles.scss.
It will be compiled along with the rest in the final style.css file.
See for example how we import other files here: https://github.com/metabrainz/bookbrainz-site/blob/master/src/client/stylesheets/style.scss#L30

You'll need to add `@import "./external-services.scss";` and  then remove mentions of the new css file from this file and from webpack.clisent.js

Reply:
This method makes more sense. I've updated it.